.macos

```diff
@@ -45,35 +48,35 @@ defaults write com.apple.systemuiserver menuExtras -array \
 	"/System/Library/CoreServices/Menu Extras/Clock.menu"
 
 # Set highlight color to green
-defaults write NSGlobalDomain AppleHighlightColor -string "0.764700 0.976500 0.568600"
+defaults write -g AppleHighlightColor -string "0.764700 0.976500 0.568600"
```

Comment:
this is superfluous 
    -g | -globalDomain | NSGlobalDomain
               Specify the global domain. '-g' and '-globalDomain' may be used as synonyms for NSGlobalDomain.

Its unlikely that this will change